packages/create/src/generators/app/header.js

```diff
@@ -0,0 +1,13 @@
+export default `
+        _.,,,,,,,,,._
+     .d''           \`\`b.
+   .p'      Open       \`q.        Open Web Components Create
+ .d'    Web Components  \`b.
+ .d'                     \`b.      It let's you scaffold or upgrade web components projects.
+ ::   .................   ::       You can call it with...
```

Comment:
This sentence seems odd to me?

Reply:
yeah this whole header is just a WIP 🙈

Reply:
I think we can just omit it

Reply:
adjusted the header - pls take another look 🤗